### ISSUE TITLE
Creating a VM from a large qcow2 image taking too long and then failing

### ISSUE THREAD

**Issue by Alex2761** [bpg/terraform-provider-proxmox#744]
### Discussed in https://github.com/bpg/terraform-provider-proxmox/discussions/737

<div type='discussions-op-text'>

<sup>Originally posted by **dandaolrian** November 24, 2023</sup>
Hi,
We have a use case where we’re deploying a windows qcow2 image of around 20Gb and were running into problems deploying to one of our nodes (others were OK)
When we took a closer look at what the provisioner we saw it was running the following commands: https://github.com/bpg/terraform-provider-proxmox/blob/a96a3a9641a5d7c387e65ea17d13ddbc82054bfd/proxmoxtf/resource/vm.go#L2790-L2796

So if we understand correctly, we copy the image, resize that import it and then throw the resized file away. 
On the Proxmox node we saw multiple qemu-img convert processes and these take a long time when importdisk is running as we do that on the resized (larger) image

We thought logically it would make more sense to import the image as is (i.e smaller) and then resize. So like this
```
`imported_disk="$(qm importdisk "$vm_id" "$source_image" "$datastore_id_target" -format $file_format | grep "unused0" | cut -d ":" -f 3 | cut -d "'" -f 1)"`,
`disk_id="${datastore_id_target}:$imported_disk${disk_options}"`,
`qm set "$vm_id" "-${disk_interface}" "$disk_id"`,
`qm resize "$vm_id" "${disk_interface}" "${disk_size}G"`,
```

With this we no longer have such long times for deployment and were down from 55mins and failing to 10mins and working 😊

We would start a pull request but weren’t sure if maybe it had been done as above for a reason and wanted to understand that before doing so
This seems to similar if not the same as https://github.com/bpg/terraform-provider-proxmox/issues/675

Incidentally when it failed we got the same error as in #675
`Post "https://dehx-pa-001:8006/api2/json/nodes/proxmox/qemu/437/status/start": context deadline exceeded`
And we never managed to track down why the start would fail. We could start manually but it failed everytime here after taking 55+ mins to create the VM

Would be interested to hear your thoughts
</div>

**Comment by bpg** [comment]
Please comment on https://github.com/bpg/terraform-provider-proxmox/discussions/737 in Discussions

**Comment by bpg** [comment]
Should be fixed by #753
